lib/features/thread/presentation/thread_controller.dart

```diff
@@ -382,13 +387,15 @@ class ThreadController extends BaseController {
   void loadMoreEmails() {
     if (canLoadMore && _accountId != null) {
       log('ThreadController::loadMoreEmails(): latest: ${emailList.last.receivedAt}');
+      startFpsMeter();
```

Comment:
how to only apply it on debug or profile mode?

Reply:
Only check `Foundation.kDebugMode` or `Foundation.kProfileMode`

Reply:
yes, please apply it from the Fps class, or here, or inside the startFpsMeter()